test/integration/targets/slurp/tasks/main.yml

```diff
@@ -16,6 +16,8 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+- include_vars: default.yml
```

Comment:
If these are default values, they should go in `defaults/main.yml` and this task can be omitted.

Reply:
I'll rename it to defaults/main.yml. Thanks!